admin-dev/themes/new-theme/js/app/utils/datepicker.js

```diff
@@ -30,9 +30,20 @@ const $ = global.$;
  * Enable all datepickers.
  */
 const init = function initDatePickers() {
-  $('.datepicker input[type="text"]').datetimepicker({
-    locale: global.full_language_code,
-    format: 'YYYY-MM-DD',
+  const datePickers = $('.datepicker input[type="text"]');
+
+  datePickers.each((key, picker) => {
```

Comment:
I think documentation for this component is needed. You can put it into docblock above.